GEON-PPANG-iOS/Presentation/NickName/NickNameViewController.swift

```diff
@@ -178,13 +185,38 @@ extension NickNameViewController: UITextFieldDelegate {
         guard let currentText = textField.text else { return false }
         let changedText = (currentText as NSString).replacingCharacters(in: range, with: string)
         
+        if currentText != changedText {
+            self.configureNextButtonUI(false)
+        }
```

Comment:
p3:
```suggestion
        self.configureNextButtonUI(currentText == changedText)
```
로 바꾸는 방법도 있을 것 같군여 ~~
근데 뭐 취향 차이 ~~

Reply:
3189ea027b7c946daa203ea07a22d8db1827be5c